src/main/java/net/yogstation/yogbot/http/byond/UnclaimedTicketEndpoint.kt

```diff
@@ -27,9 +28,10 @@ class UnclaimedTicketEndpoint(
 		val keyError = validateKey(unclaimedTicketDTO.key)
 		if (keyError != null)
 			return keyError
+		val ticketContent = StringEscapeUtils.unescapeHtml4(unclaimedTicketDTO.message)
```

Comment:
Assuming that a player can't write anything malicious in a ticket message and have it un-escaped here right?

Reply:
Discord doesn't parse html